CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 
 ## Unreleased
 
+## v0.34.0
+
+## 🚀 New components 🚀
+
+- [`cumulativetodelta` processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) to convert cumulative sum metrics to cumulative delta
+
+- [`file` exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter) from core repository (#3474)
```

Comment:
```suggestion
- [`file` exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter) from core repository ([#3474](https://github.com/open-telemetry/opentelemetry-collector/issues/3474))
```

That issue is in a different repo, so it likely won't link correctly.

Reply:
Done.